indigo_app/templates/indigo_app/place/_bulk_approve_form.html

```diff
@@ -4,7 +4,11 @@
   <div class="modal-content">
     <div class="modal-header">
       <h5 class="modal-title">
-        {% blocktrans with works_length=works_in_progress|length %}Approve {{ works_length }} works in progress{% endblocktrans %}
+        {% if works_in_progress|length > 1 %}
```

Comment:
`{% blocktrans count ... %}... {% plural %} ... {%endblocktrans %} ` will do this for you, see https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#blocktranslate-template-tag